electives/18843.md

```diff
@@ -0,0 +1,32 @@
+# 15-821/18-843: Mobile and Pervasive Computing
+
+| Category     | Difficulty |
+|:--           | :-:        |
+| Quizes       | 8          |
+| Presentation | 7          |
+| Project      | 8          |
+
+
+This course explores the principles and practice of mobile computing and its close relative, pervasive computing (aka “Internet of Things (IoT)”). The course will offer significant hands-on experience: in small teams, students will work under the guidance of a mentor on a project. Students will also, in small teams, present a summary and overview of the commercial landscape for one of the topics covered in class. There will be a brief quiz at the start of each class, based on the readings for that classes the principles and practice of mobile computing and its close relative, pervasive computing (aka “Internet of Things (IoT)”).
```

Comment:
The latter part of the last sentence doesn't make sense

Reply:
`based on the readings for that classes the principles and practice of mobile computing and its close relative, pervasive computing (aka “Internet of Things (IoT)”).`

Reply:
I will change the sentence, although that one specifically was taken from his website.